### PR TITLE
Support for Ciena service delivery switch family

### PR DESCRIPTION
**Pull request by bakerds**
- Added support for many sensors
- Added custom entity-physical discovery
- Adjusted port naming
- Fixed uptime
- Added CPU graph to device header
- Added/updated Ciena MIBs
- Added unit test data

Tested with Ciena 8700 and 5170 switches running SAOS 8.7.0 and 8.6.2.

---
DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply 11636`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/ciena-service-aggregation-switches/9187/4

File: includes/polling/os/ciena-sds.inc.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
```
Comment (murrant) [inline includes/polling/os/ciena-sds.inc.php:1]: Can you convert this file to the new format?  See LibreNMS/OS/Exa.php for an example.

**Comment by murrant** [comment]
Nice work, big PR here ;)

File: includes/polling/os/ciena-sds.inc.php (hunk 1 above)
Comment (bakerds) [reply includes/polling/os/ciena-sds.inc.php]: Yup. Done.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/poller-not-writing-mysql-or-rrd/12208/8

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-64-release-changelog-may-2020/12290/1

**Pull request merged**